src/components/oculus-touch-controls.js

```diff
@@ -150,6 +150,7 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
       controller: 0,
       rotationOffset: data.rotationOffset !== -999 ? data.rotationOffset : isRightHand ? -90 : 90
     });
+    el.emit('controller-connected', {type: 'oculus-touch-controller'});
```

Comment:
if we were to expand the `tracked-controls` schema, would the data be available when `componentinitialized` is fired for `tracked-controls` ?  or is it only available in subsequent `update` ?